src/Model/Item.php

```diff
@@ -56,6 +56,40 @@ class Item extends BaseObject
 			'author-id', 'author-link', 'owner-link', 'contact-uid',
 			'signed_text', 'signature', 'signer'];
 
+	/**
+	 * @brief Fetch a single item row
+	 *
+	 * @param mixed $stmt statement object
+	 * @return array current row
+	 */
+	public static function fetch($stmt)
+	{
+		$row = dba::fetch($stmt);
+
+		return $row;
+	}
+
+	/**
+	 * @brief Fills an array with data from an item query
+	 *
+	 * @param object $stmt statement object
+	 * @return array Data array
+	 */
+	public static function inArray($stmt, $do_close = true) {
```

Comment:
This function seems to be the exact copy of `dba::inArray`, what is its purpose?

Reply:
In the near future this function will do some changes to the queried item lines. We will move the body to a new table, but will keep it on the old place for old item entries. This function will then separate between the different values.

Also it will be used to always show the best author name. For feed users we have to use the data from the contact, but for items without a matching contact we have to use the data from the public contact.